frontend-mini-challenge/components/LoggedInPage.tsx

```diff
@@ -16,12 +16,25 @@ import { useRouter } from "next/router";
 import { clearAuthToken } from "../lib/authTokenStorage";
 import { useCookies } from "react-cookie";
 
+interface Props {
```

Comment:
The different components have different props. Please use different interfaces for the different things.

If the foo function has the arguments a, b and c and the bar function has d, e and f then it is incorrect to say that foo requires a,b,c,d,e, f, req and query. 

Please make use of the correct interface. It should tell the truth of what is required in each function.. Each function requires different things. We need to be exact and specific. If you add in extra things that aren't needed at all then that makes it worse.

Reply:
please also make a point of importing things that are defined elsewhere in the code.

Reply:
and req is definitely not a string.